bluebottle/settings/base.py

```diff
@@ -145,6 +145,7 @@
     'bluebottle.auth.middleware.AdminOnlySessionMiddleware',
     'bluebottle.auth.middleware.AdminOnlyCsrf',
     'bluebottle.auth.middleware.AdminOnlyAuthenticationMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
This also requires a change to the settings generated in server-deployment right?